s3-source-connector/src/main/java/io/aiven/kafka/connect/s3/source/utils/RecordProcessor.java

```diff
@@ -44,9 +44,9 @@ private RecordProcessor() {
 
     }
     public static List<SourceRecord> processRecords(final Iterator<List<AivenS3SourceRecord>> sourceRecordIterator,
-            final List<SourceRecord> results, final S3SourceConfig s3SourceConfig,
-            final Optional<Converter> keyConverter, final Converter valueConverter,
-            final AtomicBoolean connectorStopped, final OutputWriter outputWriter, final Set<String> failedObjectKeys) {
+                                                    final List<SourceRecord> results, final S3SourceConfig s3SourceConfig,
```

Comment:
Can you run` ./gradlew :s3-source-connector:spotlessApply` I think the formatting looks funny